components/automate-ui/src/styles/_colors.scss

```diff
@@ -2,6 +2,42 @@
 // ------------------------------------------------------------------------------------- //
 // DO NOT DIRECTLY IMPORT COLORS FROM THIS FILE : CREATE A VARIABLE in _variables.scss //
 // ------------------------------------------------------------------------------------- //
+$white: #FFFFFF;
+$eggplant: #394261;
+$lavender: #BBCDF2;
+$eggplant-light-1: #C5C7D1;
+$eggplant-light-2: #B1B4C1;
+$eggplant-light-3: #9DA1B1;
+$eggplant-light-4: #898EA1;
+$eggplant-light-5: #757B91;
+$eggplant-light-6: #616881;
+$eggplant-light-7: #4D5571;
+$eggplant-dark-1: #333B57;
+$eggplant-dark-2: #2D344D;
+$lavender-light-1: #ECF0F9;
+$lavender-light-2: #E5EBF8;
+$lavender-light-3: #DEE6F7;
+$lavender-light-4: #D7E1F6;
+$lavender-light-5: #D0DCF5;
+$lavender-light-6: #C9D7F4;
+$lavender-light-7: #C2D2F3;
+$lavender-dark-1: #A8B8DA;
+$lavender-dark-2: #95A3C2;
+$grape-light-1: #D0D4EE;
+$grape-light-2: #BFC5E8;
+$grape-light-3: #AEB6E2;
+$grape-light-4: #9DA7DC;
+$grape-light-5: #8C98D6;
+$grape-light-6: #7B89D0;
+$grape-light-7: #6A7ACA;
+$grape-dark-1: #5060B0;
+$grape-dark-2: #47559C;
+$cornflower: #3023AE;
+$orchid: #C86DD7;
+$daylily: #FEA900;
+$crimson: #E71D36;
+$rose: #E61F36;
+
```

Comment:
Curious where these vars came from. Carbon?

Reply:
colors came from liz's designs, they were defined in https://github.com/chef/chef-design-system/issues/4

we still need to map these to the carbon design system variables, that will hopefully be done this week